components/ui/AppDataTable.vue

```diff
@@ -1,16 +1,26 @@
 <template>
-  <cv-data-table class="app-data-table" :columns="columns" :sortable="false">
-    <template #data>
+  <bx-table class="app-data-table" :columns="columns">
```

Comment:
Do we still need `:columns="columns"` here?

Reply:
Removed in https://github.com/Qiskit/qiskit.org/pull/3001/commits/2a16a8f02b5cad70e110824ea300023f74d90b99.